src/main/java/com/redhat/cloud/notifications/routers/NotificationService.java

```diff
@@ -365,17 +366,14 @@ private String getAddress(Principal principal) {
     public Uni<List<BehaviorGroup>> findBehaviorGroupsByBundleId(@Context SecurityContext sec, @PathParam("bundleId") UUID bundleId) {
         return getAccountId(sec)
                 .onItem().transformToUni(accountId -> behaviorGroupResources.findByBundleId(accountId, bundleId))
-                .onItem().call(behaviorGroups -> behaviorGroups.size() == 0 ? Uni.createFrom().voidItem() : Uni.combine().all().unis(
-                        behaviorGroups.stream()
+                .onItem().call(behaviorGroups -> endpointResources.loadProperties(
+                        behaviorGroups
+                                .stream()
                                 .map(BehaviorGroup::getActions)
-                                .map(bga -> bga.size() == 0 ? Uni.createFrom().voidItem() : Uni.combine().all().unis(
-                                        bga.stream()
-                                                .map(BehaviorGroupAction::getEndpoint)
-                                                .map(endpoint -> endpointResources.loadProperties(endpoint))
-                                                .collect(Collectors.toList())
-                                        )
-                                                .discardItems()
-                                ).collect(Collectors.toList())
-                ).discardItems());
```

Comment:
I just saw while working on something else that there is a concurrency issue here. `Uni.combine()` does not work well with the Hibernate Reactive session because `combine` will cause concurrent execution while the Hibernate session must not be used concurrently.

I'm not fixing it since it will soon be replaced with something else.